### DISCUSSION TITLE
Publishing WFS and WMS

### DISCUSSION THREAD

**Discussion by gi921120** [opengisch/QField#2605] [Ideas]
Hi all, 

It would be great to work with some layers as WFS or WMS. I believe that this will prevent the app from crashing and improve the rendering speed when the data is too large or heavy. 

Kind regards, 

Giovanny

**Comment by braaksman** [comment]
Agreed, it would be good to have QField Cloud accept layers within QField that are accessed via WFS. This would:
-  Remove the need for the database to be publicly accessible.
- Compared to Postgresql, using software such as Geoserver makes it easy to manage user credentials, read and write access to the database. 
Thanks
